ARSN-22 - Fix bug in `onPutObjectTagging` and `getObjectTagging`

lavigne958 opened the pull request:
do not use the version stored in the location object,
use the object version located in the object metadata.

quick fix on the way, use actual string instead of object
when building object key (for the bucket part).

Commit:
ARSN-22 - Fix bug in `onPutOgjectTaggig` and `getObjectTagging`

do not use the version stored in the location object,
use the object version located in the object metadata.

quick fix on the way, use actual string instead of object
when building object key (for the bucket part).

bert-e commented:
# Hello lavigne958,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/arsenal/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ARSN-22 contains:


* `8.1.3`



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `8.1.4`


Please check the `Fix Version/s` of ARSN-22, or the target
branch of this pull request.

lavigne958 commented:
ping

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

Commit:
ARSN-22 - Add unit tests on put/delete object tagging.

Added new functions `deleteObjectTagging` and `putObjectTagging`
in the dummy service for unit tests.

Add 2 new tests in unit tests for external back-ends:

1. test `putObjectTagging` and `deleteObjectTagging`
2. test the same functions but with a version ID.

lavigne958 commented:
/codecov

bert-e commented:
# Unknown command



I didn't understand this comment by @lavigne958:

> /codecov

I don't know what `codecov` means.

Please **edit** or **delete** the corresponding comment so I can move on.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

lavigne958 commented:
close this PR, need to re-work it. this is not the correct fix.
